resolwe/process/fields.py

```diff
@@ -501,8 +501,11 @@ def __init__(self, data_id, field, cache):
         """
         super().__setattr__("_data_id", data_id)
 
+        if not cache:
```

Comment:
You should set the default value of the `cache` to `None` if it is not required.

Reply:
Done.